### PR TITLE
Exercise 1: styling the blog

### PR DESCRIPTION
**Pull request by karlsc**

### CODE REVIEW

File: blog/scss/main.scss [hunk 1]
```diff
@@ -1 +1,131 @@
...
\ No newline at end of file
+body {
+    margin:0;
+    
+}
+
+
+.main-header {
+    
+    overflow:hidden;
```
Comment (nyanofthemoon) [inline blog/scss/main.scss]: @karlysun Is this rule needed?

**Comment by karlysun** [comment]
It seems like we do not. Updated

File: blog/scss/main.scss [hunk 2]
```diff
@@ -1 +1,128 @@
...
+        &::before {
+            content:">>";
+            color:hsla(254, 85%, 25%, 0.99);
+            background-color:black;
+            border-radius:10%;
+            padding:0 0.3em 0 0.3em;
+            margin-right:0.2em;
+            box-shadow:2px 1px 10px black;
+            display:inline-block;
+            font-size:20px;
```
Comment (nyanofthemoon) [inline blog/scss/main.scss]: @karlysun I try and use px for font sizes at the root and em for components.

File: blog/scss/main.scss [hunk 3]
```diff
@@ -1 +1,128 @@
...
+        
+        &:not(&--first) {
+            text-indent:3em;
+        }
+        
+        &--first {
+            margin-top:0;
+            
+            &::first-letter {
+                font-size:70px;
```
Comment (nyanofthemoon) [inline blog/scss/main.scss]: @karlysun I try and use px for font sizes at the root and em for components.

**Comment by karlysun** [comment]
@nyanofthemoon updated

**Pull request closed without merging**